exercises/concept/paolas-prestigious-pizza/.meta/Exemplar.elm

```diff
@@ -0,0 +1,92 @@
+module PaolasPrestigiousPizza exposing
+    ( Pizza
+    , ingredientsParser
+    , menuParser
+    , oneIngredientParser
+    , pizzaParser
+    , priceParser
+    , vegetarianParser
+    , wordParser
+    )
+
+import Parser exposing ((|.), (|=), Parser)
+
+
+type alias Pizza =
+    { name : String
+    , vegetarian : Bool
+    , ingredients : List String
+    , price : Int
+    }
+
+
+priceParser : Parser Int
+priceParser =
+    Parser.int
+        |. Parser.symbol "€"
+
+
+vegetarianParser : Parser Bool
+vegetarianParser =
+    Parser.oneOf
+        [ Parser.map (always True) (Parser.symbol "(v)")
+        , Parser.succeed False
+        ]
+
+
+wordParser : Parser String
+wordParser =
+    Parser.chompWhile Char.isAlpha
+        |> Parser.getChompedString
+
+
+ingredientsParser : Parser (List String)
+ingredientsParser =
+    sequenceParser "," oneIngredientParser
+
+
+pizzaParser : Parser Pizza
+pizzaParser =
+    Parser.succeed Pizza
+        |= wordParser
+        |. Parser.spaces
+        |= vegetarianParser
+        |. Parser.spaces
+        |. Parser.symbol ":"
+        |. Parser.spaces
+        |= ingredientsParser
+        |. Parser.spaces
+        |. Parser.symbol "-"
+        |. Parser.spaces
+        |= priceParser
+
+
+menuParser : Parser (List Pizza)
+menuParser =
+    sequenceParser "" pizzaParser |. Parser.end
+
+
+oneIngredientParser : Parser String
+oneIngredientParser =
+    Parser.chompWhile (\c -> Char.isAlpha c || c == ' ')
```

Comment:
This coudl reuse wordParse, not sure if it is a good idea or not though ...

Reply:
In a sequence or a loop? I think this is simpler